app/src/ui/preferences/appearance.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react'
+//import { Row } from '../lib/row'
```

Comment:
will we be using this in a later commit or is this old cruft?

Reply:
Copy paste mistakes